### PR TITLE
upgrade omnipay/common from 2.0 to 3.0

### PR DESCRIPTION
**Pull request by alcalyn**
I can't make it work with newer version of omnipay/common, required as omnipay 2.0 uses old versions of many dependencies.

### CODE REVIEW

**Comment by aschelch** [comment]
Looks good ! Well done ;)

File: tests/GatewayTest.php [hunk 1]
```diff
@@ -18,15 +18,15 @@ public function testPurchase()
         $request = $this->gateway->purchase(array('amount' => '10.00'));
 
         $this->assertInstanceOf('Omnipay\PayZen\Message\PurchaseRequest', $request);
-        $this->assertSame('1000', $request->getAmount());
+        $this->assertSame('10.00', $request->getAmount());
```
Comment (aschelch) [inline tests/GatewayTest.php]: Hmm wait, why did you change that ?

Comment (alcalyn) [reply tests/GatewayTest.php]: because it seems that in Omnipay v3, the format of amount changed. But it does not impact the format of amount which is used when redirected to Payzen page: I still have i.e `"vads_amount" => "5500"` for 55€.
